tools/codegen/codespec/merge_attributes.go

```diff
@@ -6,7 +6,7 @@ func mergeAttributes(pathParams, createRequest, createResponse, readResponse Att
 	merged := make(map[string]*Attribute)
 
 	addOrUpdate := func(attr *Attribute, computability ComputedOptionalRequired) {
-		if existingAttr, exists := merged[attr.Name]; exists {
+		if existingAttr, exists := merged[attr.Name.SnakeCase()]; exists {
```

Comment:
curious why is this required?

Reply:
found it useful to use a Go type definition over a regular string, this way when you access the value of Name you do it through different functions which are explicit on which string format you will receive.